tests/dummy/app/pods/different-imports/template.hbs

```diff
@@ -0,0 +1,31 @@
+// spaced
+{{ import Button from "ui/button" }}
+// tabbed
+{{  import    LocalButton from    "application/button"    }}
+// mixed
+{{  import            ButtonButton           from          "ui/button"  }}
+// newlined
+{{import 
+  LocalButtonLocalButton 
+  from
+  "application/button"
+}}
+// newlined mixed
+{{import 
+  ButtonButtonButton             
+  from           
+  "ui/button"   
+}}
+// newlined mixed with tabs and spaces
+{{  import 
+  LocalButtonLocalButtonLocalButton    
+  from           
+  "application/button"                  
+}}
+
+<Button>b1</Button>
+<LocalButton>b1</LocalButton>
```

Comment:
Same as above - I think it's a little clearer if we refer to these as "relative" and "absolute" imports, rather the "local" and "global" - the "global" might lead to some confusion that it is somehow a "global variable".

Reply:
`LocalButton` renamed to `SecondButton`, because it's absolute and no difference for mutiline import is it relative or absolute.